### PR TITLE
Update schema definitions and add new ones.

### PR DESCRIPTION
**Pull request by faustow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,18 +20,10 @@ your root schema using `allOf` keyword:
...
-  package. It is required to let recipient know what's inside the package.
-- `$originAddress`: original address of the participant that created
-  the data within this data package. This should remain unchanged during
-  the repackaging process so that the final recipient would know who
-  originally created this data (rather than the address of the user who
-  repackaged the data)
-- `$recipientAddress`: address of the designated recipient. It is needed
-  to ensure the recipient that the data was indeed designated for him.
-- `$createdAt`: time value that denotes when original data was created.
-  This should remain unchanged during the repackaging process to let the
```
Comment (mclate) [inline README.md:38]: Why did you remove info about persistency of the fields during repackaging?

File: basicUserProfile.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "$id": "https://getdock.github.io/schemas/basicUserProfile.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Basic user profile.",
+  "title": "Basic User Profile",
+  "allOf": [
+    {
+      "$ref": "https://getdock.github.io/schemas/dock.json"
+    },
```
Comment (mclate) [inline basicUserProfile.json]: Missing `]`

File: education.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  ],
+  "definitions": {
+    "education": {
+      "type": "object",
+      "properties": {
+        "school": {
+          "type": "object",
+          "properties": {
+            "aliases": {
+              "type": "array",
```
Comment (mclate) [inline education.json:19]: Do we  want to add `"uniqueItems": true` property?

File: email.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "$id": "https://getdock.github.io/schemas/email.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "User email.",
+  "title": "User email",
+  "allOf": [
+    {
+      "$ref": "https://getdock.github.io/schemas/dock.json"
+    },
```
Comment (mclate) [inline email.json]: Missing `]`

File: basicUserProfile.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  "type": "object",
+  "properties": {
```
Comment (mclate) [inline basicUserProfile.json]: These properties should be within `$data` object. This applies for all schemas updated here.

```json
"type": "object",
"properties": {
    "$data": {
        "type": "object",
        "properties": {
             "firstName": {....}
        }
    }
}
```

File: jobRecommendations.json [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+          "type": "string"
+        },
+        "authorAvatar": {
+          "type": "string"
+        },
+        "source": {
+          "type": "string",
+          "description": "Platform on which this recommendation was created on."
+        },
+        "createdAt": {
```
Comment (mclate) [inline jobRecommendations.json:31]: There is a predefined data format in json schema: 
```json
"createdAt": {
    "type": "string",
    "format": "date-time"
}
```
which is RFC3339 (sort-of ISO format)

File: skills.json [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "years": {
+          "type": "number"
+        },
+        "level": {
+          "type": "integer"
```
Comment (mclate) [inline skills.json]: Think we should define boundaries here. Otherwise it will be hard for other tools to make sense of unbounded integer (is it 5 stars? or is it 1-10 grade? or is it percentage?)

File: userContactInfo.json [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+      "description": "User's country",
+      "type": "string"
+    },
+    "city": {
+      "description": "User's city",
+      "type": "string"
+    },
+    "phone": {
+      "description": "User's phone number",
+      "type": "number"
```
Comment (mclate) [inline userContactInfo.json]: Don't know if this should be a number. Current representation eliminates all phones that have extensions.

File: userProfile.json [hunk 9]
```diff
@@ -1,187 +1,21 @@
...
-          "type": "string"
-        },
-        "short_name": {
-          "type": "string"
-        },
-        "types": {
-          "$ref": "#/definitions/types"
-        }
-      }
     },
```
Comment (mclate) [inline userProfile.json]: Missing `]`

File: skills.json (hunk 7 above)
Comment (faustow) [reply skills.json]: Sounds good, what would be a good set of boundaries?

File: README.md (hunk 1 above)
Comment (faustow) [reply README.md:38]: This belongs to some other document. But adding it back for now, until we have that other doc.

File: email.json (hunk 4 above)
Comment (mclate) [reply email.json]: according to json format you can't have trailing comma at the end of the list.

File: basicUserProfile.json (hunk 2 above)
Comment (mclate) [reply basicUserProfile.json]: According to json format you can't have trailing comma at the end of the list.

**Review by mclate — APPROVED** [review]
